src/main/java/uk/gov/hmcts/divorce/divorcecase/model/RetiredFields.java

```diff
@@ -18,6 +18,9 @@
 @NoArgsConstructor
 public class RetiredFields {
 
+    @CCD(label = "Retired solServiceMethod")
+    private ServiceMethod solServiceMethod;
```

Comment:
Maybe move this to line 30 below: `private Set<ThePrayer> applicant2PrayerHasBeenGivenCheckbox;` and remove the `@CCD` annotation as it's not needed really.